doc/source/user/absolute_beginners.rst

```diff
@@ -929,18 +960,30 @@ array. NumPy offers functions like ``ones()`` and ``zeros()``, and the
 All you need to do is pass in the number of elements you want it to generate::
 
   >>> np.ones(3)
+  array([1., 1., 1.])
   >>> np.zeros(3)
-  >>> np.random.random(3)  # the simplest way to generate random numbers
+  array([0., 0., 0.])
+  # the simplest way to generate random numbers
+  >>> np.random.default_rng().random(3)
+  array([0.08419554, 0.01447087, 0.88581866])  # may vary
 
 .. image:: images/np_ones_zeros_random.png
 
 You can also use ``ones()``, ``zeros()``, and ``random()`` to create
 an array if you give them a tuple describing the dimensions of the matrix::
```

Comment:
```suggestion
a 2D array if you give them a tuple describing the dimensions of the matrix::
```
Otherwise this is just repeating the sentence above.

Reply:
ok